pkg/sdk/poc/generator/keyword_builders.go

```diff
@@ -79,7 +79,7 @@ func (v *QueryStruct) SetTags() *QueryStruct {
 }
 
 func (v *QueryStruct) OptionalSetTags() *QueryStruct {
-	return v.setTags(nil)
+	return v.setTags(KeywordOptions().SQL("SET TAG"))
```

Comment:
I think it is also needed for SetTags and UnsetTags (but I don't get what has hanged in the meantime, because these were working properly).